src/compiler/compile/render_dom/wrappers/InlineComponent/index.ts

```diff
@@ -106,11 +106,28 @@ export default class InlineComponentWrapper extends Wrapper {
 		block.add_outro();
 	}
 
+	warn_if_reactive() {
+		const { name } = this.node;
+		const variable = this.renderer.component.var_lookup.get(name);
+		if (!variable) {
+			return;
+		}
+
+		if (variable.reassigned || variable.export_name || variable.is_reactive_dependency) {
+			this.renderer.component.warn(this.node, {
+				code: 'reactive-component',
+				message: `<${name} /> will not be reactive. Use <svelte:component this={${name}} /> if you want reactivity.`,
```

Comment:
This makes it sound like `<Foo/>` isn't reactive at all, when we should just be telling the user that it won't change when `Foo` changes. I'm not sure how to word this.

Reply:
Maybe '<Foo/> will not be reactive if Foo changes. Use <svelte:component this={Foo}/>' if you want this reactivity.'?